src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -96,7 +116,40 @@ export default function ApplicationInformation(props) {
           : div({className: 'text-placeholder', key: 'non-technical-summary-placeholder', style: { height: '18rem',
             width: '100%',
           }})
-      ])
+      ]),
+      div({className: 'application-details-container', style: { margin: '2.5rem 0'}}, [
+        div({className: 'applicant-details-subheader', style: styles.title}, ["Application Details"]),
+        div({className: 'information-row', style: styles.row}, [
+          generateLabelSpanContents('External Collaborators', 'external-collaborators', externalCollaborators, isLoading),
+          generateLabelSpanContents('Internal Collaborators', 'internal-collaborators', internalCollaborators, isLoading)
+        ]),
+        div({className: 'information-row', style: styles.row}, [
+          generateLabelSpanContents('Signing Official', 'signing-official', signingOfficial, isLoading),
+          generateLabelSpanContents('IT Director', 'it-director', itDirector, isLoading)
+        ]),
+        div({className: 'information-row', style: styles.row}, [
+          generateLabelSpanContents('Signing Official Email', 'signing-official-email', signingOfficialEmail, isLoading),
+          generateLabelSpanContents('IT Director Email', 'it-director-email', itDirectorEmail, isLoading),
+        ]),
+        div({className: 'information-row', style: styles.row}, [
+          generateLabelSpanContents('Internal Lab Staff', 'internal-lab-staff', internalLabStaff, isLoading),
+          generateLabelSpanContents('Using AnVIL only for storage and analysis', 'anvil-storage', anvilStorage, isLoading),
+        ]),
+        div({className: 'information-row', style: styles.row}, [
```

Comment:
Whoops, I had added a new style _applicantInfoRow_ but I implemented it in the Applicant Information section instead of the Application Details by accident. Correcting this on my next commit